DSharpPlus/DebugLogger.cs

```diff
@@ -66,5 +65,10 @@ public class DebugLogMessageEventArgs : EventArgs
         public string Application;
         public string Message;
         public DateTime TimeStamp;
+
+        public string ToString()
+        {
+            return $"[{Application.ToUpper()}] {TimeStamp} | {Level.ToString().ToUpper()}: {Message}";
+        }
```

Comment:
I would change the format so that it is `$"[{TimeStamp.ToString("yyyy-MM-dd HH:mm:ss zzz")}] [{Application}] [{Level}] {Message}"` for clarity and consistency. While at it, update any other occurence.